internal/component/otelcol/receiver/kafka/kafka_test.go

```diff
@@ -433,3 +429,32 @@ func TestDebugMetricsConfig(t *testing.T) {
 		})
 	}
 }
+
+func TestArguments_Validate(t *testing.T) {
+	cfg := `
+		brokers = ["10.10.10.10:9092"]
+		protocol_version = "2.0.0"
+		topic = "traces"
+		output {
+		}
+	`
+	var args kafka.Arguments
+	require.NoError(t, syntax.Unmarshal([]byte(cfg), &args))
+
+	// Adding two traces consumer, expect no error
+	tracesConsumer := fakeconsumer.Consumer{}
+	args.Output.Traces = append(args.Output.Traces, &tracesConsumer)
+	tracesConsumer2 := fakeconsumer.Consumer{}
+	args.Output.Traces = append(args.Output.Traces, &tracesConsumer2)
```

Comment:
tiny nit: since the consumers aren't used after the append, the test might be slightly more readable if they were defined inline to the append:

```
args.Output.Traces = append(args.Output.Traces, &fakeconsumer.Consumer{}) 
args.Output.Traces = append(args.Output.Traces, &fakeconsumer.Consumer{}) 
```

(ditto with the other lines below)